tests/lib.rs

```diff
@@ -11,7 +11,8 @@ use hipchat_client::Client as HipchatClient;
 struct Config {
     token: String,
     origin: String,
-    room: String
+    room: String,
+    user: String
```

Comment:
When trying to run the integration tests, I initially found myself use my username for the configuration, which would not work.

To make the intention more clear, could we update this type to be a number, and update the example.settings.json?